.gitlab-ci.yml

```diff
@@ -21,6 +20,7 @@ test:
   script:
     - make extracheck
     - make coverage
+    - make build V=1
```

Comment:
I think we should test just `make` instead. This is what is typically used to build.

Reply:
Agreed :+1: 

`make build V=1` was what we were using in Travis